clinica/pipelines/pet_surface/pet_surface_utils.py

```diff
@@ -178,7 +178,7 @@ def isclose(a, b, rel_tol=1e-9, abs_tol=0.0):
     # Read label from gtmsegfile, change data into integers in order to have no problems when testing equality of labels
     label = nib.load(gtmsegfile)
     label.header.set_data_dtype("int8")
-    volume = label.get_fdata()
+    volume = label.get_fdata(dtype="float32")
```

Comment:
There is something weird going on here. The comment says we want to load the data as integers, yet we load the volume as float.